### DISCUSSION TITLE
Cannot toggle ready when in Results

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#16230] [Q&A] [answered]
Self explanatory, but i'm not in results. I don't think i did anything weird between gameplay and returning to lobby.

![image](https://user-images.githubusercontent.com/53081750/147284366-b4515586-745a-43a8-aa36-b1e423cb14e5.png)

error at 2021-12-23 19:26:09
[network.log](https://github.com/ppy/osu/files/7771077/network.log)
[performance.log](https://github.com/ppy/osu/files/7771074/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7771075/runtime.log)
[database.log](https://github.com/ppy/osu/files/7771076/database.log)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/16236
